modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm

```diff
@@ -379,6 +379,46 @@ But i keeped it as unobtainable breain trauma, so admins can add it through VV *
 			return TRUE
 	return FALSE
 
+//Shibari update quirks: Rope bunny and rigger. One have additional mood bonus (0) and exist for same reason as ananas affinity, other one can faster tie ropes on character because why not.
+//Rope bunny code
+/datum/quirk/ropebunny
```

Comment:
Nope.
We shouldn't have quirks that are purely for ERP purposes in the quirk menu. I suggest we just add a kink selection menu for users to choose from. There's enough random ERP related bloat in t he quirk menu as it stands, it's long overdue for a rework to not be using the quirk system so that we can properly hide ERP quirks from the UI for users who don't want to see them via having them disabled in the options.

Reply:
Quirk lists and menu’s are actually a good idea, but it’s too hard to implement as feature by our team rn

Reply:
Well if you're not going to refactor the stuff to not use quirks, then just don't use quirks.

Reply:
Excuse me, but this is becoming a joke now. First you wanted us to implement GAGS system, it took time, we reworked some code, added new features, and finished it looking really good. Now you want us to refactor your quirks code. 

We don't have enough experience for working with UI, and removing rigger quirk will be hard, because it's used in many parts of our code and we'll be supposed to rewrite it, fix bugs, run tests and fix bugs again, another month of working on this small update.

Reply:
By the way, we already want to rework all ERP quirks, but later. Atomizing reasons. Current PR is already too complicated for an update that contains just 2 new items and 1 new furniture object.